pkg/minikube/image/image.go

```diff
@@ -49,6 +49,12 @@ var defaultPlatform = v1.Platform{
 	OS:           "linux",
 }
 
+// UseDaemon is if we should look in local daemon for image ref
+var UseDaemon = true
```

Comment:
Why public ? Private if not used outside

Reply:
Added public method